CI: Post Merge Report | breaking change to Label Workflow

## Post Merge Report
2026-08-20 16:46:00 UTC

### Expected
```
gh: To use GitHub CLI in a GitHub Actions workflow, set the GH_TOKEN environment variable. Example:
  env:
    GH_TOKEN: ${{ github.token }}
Error: Process completed with exit code 4.
##[debug]Finishing: Build PR list
```

- Note: there will be some breaking changes from this PR to Resolve

_Originally posted by @xsscx in https://github.com/InternationalColorConsortium/iccDEV/issues/2201#issuecomment-5358964681_

--- a/.github/workflows/update-labels.yml
+++ b/.github/workflows/update-labels.yml
@@ -80,7 +80,8 @@ jobs:
         shell: bash --noprofile --norc {0}
         env:
           BASH_ENV: /dev/null
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_TOKEN: ${{ github.token }}
+          GH_REPOSITORY: ${{ github.repository }}
           EVENT_PR_NUMBER: ${{ github.event.pull_request.number }}
         run: |
           set -euo pipefail
@@ -139,7 +140,7 @@ jobs:
         shell: bash --noprofile --norc {0}
         env:
           BASH_ENV: /dev/null
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_TOKEN: ${{ github.token }}
           GH_REPO: ${{ github.repository }}
           PR_LIST: ${{ steps.pr_list.outputs.prs }}
         run: |

